tests/backup/backup_test.go

```diff
@@ -172,17 +172,20 @@ var _ = Describe("{BasicBackupCreation}", func() {
 		}
 	})
 	It("Basic Backup Creation", func() {
+		timestamp := strconv.Itoa(int(time.Now().Unix()))
+		PreRuleName := fmt.Sprintf("%s-%s", "backup-pre-rule", timestamp)
```

Comment:
Will it make much more sense to put in app name at start like `<AppName>-backup-pre-rule-<TimeStamp>`?

Reply:
Yes correct.
Here it is used just to verify the helper method "CreateRuleForBackup".
In real test case when we pass the rule name, will give like <AppName>-backup-pre-rule-<TimeStamp> .